observabilityclient/v1/deploy.py

```diff
@@ -86,7 +86,9 @@ def take_action(self, parsed_args):
                           f'{name} at port {port}', end='')
                 node = _curl(host, port, timeout=1)
                 if node:
-                    endpoints.setdefault(service.strip(), []).append(node)
+                    endpoints.setdefault(host["host"], dict()).setdefault(
```

Comment:
I think this change was to accommodate my incorrect suggestion about where prometheus would be deployed, so needs to be undone.

Reply:
Not necessarily. It will enable us implement scrape sharding in future if we will need it. Having the information on which host the endpoint is located does not hinder from anything (at least I don't see it).

Reply:
Yes, for sure I see the value in the other arrangement as well, I understand the value for flexibility like sharding, and don't see it as hindering anything.  But the code didn't  work when I ran it, it gave

('cannot represent an object', 'overcloud-compute-0')

So I reverted it to a working state.

Reply:
Ok, fair enough. Also when I'm working on the config creation now the per-host arrangement makes template creation unnecessary complex, so reverted.